File: 1._Belajar_Python/1._Latihan_Variasi/Latihan_Error_Handling_A.py
# Error Handling

print ("\n Error Handling \n")


try:
    hasil = 10 / 0
    print (hasil)
    
except ZeroDivisionError:
    print ("Gagal")
    
else:
    print ("Mantap jiwa")
    
finally:
    print ("Selesai dong")
    
    
print ("\n --- Batas --- \n")




try:
    hasil = "Tes Aja"
    print (hasil)
    
except SyntaxError:
    print ("Gagal")
    
else:
    print ("Mantap jiwa")
    
finally:
    print ("Selesai dong")

File: 1._Belajar_JavaScript_Console/1._Latihan_Variasi/Latihan_Error_Handling_A.js
// Error Handling

console.log ("\n Error Handling \n")


try {
    var hasil = "Tes"
    console.log (hasil)
}

catch (SyntaxError) {
    console.log ("Gagal")
}

finally {
    console.log ("Selesai dong")
}


console.log ("\n --- Batas -- \n")




try {
    var hasil = 10 / 0
    console.log (hasil)
}

catch (SyntaxError) {
    console.log ("Gagal")
}

finally {
    console.log ("Selesai dong")
}


console.log ("\n --- Batas -- \n")




try {
    hasil = 10 + 10 * 10
    console.log (hasil)
} 

catch (SyntaxError) {
    console.log ("Gagal")
}

finally {
    console.log ("Selesai dong")
}
